NMS-11810: Database Reports documentation (#4663)
OpenNMS/opennms: docs/modules/operation/pages/database-reports/templates/morning.adoc

[[morning-report]]
= Early Morning Report

== Report Contents

This report provides a global overview of outages, notifications, and events in the last 24 hours.

In the Outage Overview section, there are two pie charts, one that shows the current outages and one that shows the outages resolved in the last 24 hours.
// QUESTION: Not sure what the pie slices represent

The next section provides outage details in two tables, the first shows the current outages by nodes and the second table shows the resolved outages by node.

The third section is a stacked bar chart that shows teh resolved outages over the last 24 hours.
//QUESTION: Not sure what each stack represents.

> The fourth section provides an overview of notifications over the past 24 hours.  It includes a pie chart and table showing the number of notifications by the media type.

Reviewer: ```suggestion
The fourth section provides an overview of notifications over the past 24 hours.
It includes a pie chart and table showing the number of notifications by the media type.
```

// QUESTION: I don't know what the slices are.

// QUESTION: I am not sure what "media type"

> The fifth section provides and overview of events over the last 24 hours.  The first stacked bar chart shows the types of events in each hour.  The second stacked bar chart shoes the top 20 nodes with events and includes all events acknowledged and unacknowledged.

Reviewer: ```suggestion
The fifth section provides and overview of events over the last 24 hours.
The first stacked bar chart shows the types of events in each hour.
The second stacked bar chart shoes the top 20 nodes with events and includes all events acknowledged and unacknowledged.
```

[[morning-report]]
= Early Morning Report

> == Report Contents

Reviewer 2: ```suggestion

```

This report provides a global overview of outages, notifications, and events in the last 24 hours.

In the Outage Overview section, there are two pie charts, one that shows the current outages and one that shows the outages resolved in the last 24 hours.
// QUESTION: Not sure what the pie slices represent

> The next section provides outage details in two tables, the first shows the current outages by nodes and the second table shows the resolved outages by node.

Reviewer 2: ```suggestion
The next section provides outage details in two tables; the first shows the current outages by nodes and the second shows the resolved outages by node.
```

> The third section is a stacked bar chart that shows teh resolved outages over the last 24 hours.

Reviewer 2: ```suggestion
The third section is a stacked bar chart that shows the resolved outages over the last 24 hours.
```

//QUESTION: Not sure what each stack represents.

The fourth section provides an overview of notifications over the past 24 hours.
It includes a pie chart and table showing the number of notifications by the media type.
// QUESTION: I don't know what the slices are.

// QUESTION: I am not sure what "media type"

The fifth section provides and overview of events over the last 24 hours.
The first stacked bar chart shows the types of events in each hour.
The second stacked bar chart shoes the top 20 nodes with events and includes all events acknowledged and unacknowledged.
// QUESTION:  It is not clear what the stacks are, nor is it clear how the second stacked bar chart is different from the first.

> This report can be customized with a company logo in the page header that is approximately 195px by 50 px.

Reviewer 2: ```suggestion
You can customize this report with a company logo (approximately 195px by 50px) in the page header.
```

== Parameters

[options="header, autowidth"]
[cols="1,2,3,4"]

|===
| Field
| Type
| Description
| Default

| Report Format
| Choice
| Dropdown menu with a choice of PDF or CSV
| PDF

| Deliver the Report
| Checkbox

> | Check the box to deliver the report via email upon report creation

Reviewer 2: ```suggestion
| Check the box to deliver the report via email upon report creation.
```

| Unchecked

| Schedule the Report
| Checkbox

> | Check this box to create a schedule for the report

Reviewer 2: ```suggestion
| Check this box to create a schedule for the report.
```

[[morning-report]]
= Early Morning Report

This report provides a global overview of outages, notifications, and events in the last 24 hours.

In the Outage Overview section, there are two pie charts, one that shows the current outages and one that shows the outages resolved in the last 24 hours.
The slices of each pie represent the percent of the four types of outages: nodes, interfaces, services, and management.

The next section provides outage details in two tables: the first shows the current outages by nodes and the second shows the resolved outages by node.

The third section is a stacked bar chart that shows the current outages stacked on the resolved outages over the last 24 hours.

The fourth section provides an overview of notifications over the past 24 hours.
It contains a pie chart and a table showing how the notifications were received, for example: email, chat, etc.

> The fifth section provides and overview of events over the last 24 hours.

Reviewer 2: ```suggestion
The fifth section provides an overview of events over the last 24 hours.
```

[[morning-report]]
= Early Morning Report

This report provides a global overview of outages, notifications, and events in the last 24 hours.

In the Outage Overview section, there are two pie charts, one that shows the current outages and one that shows the outages resolved in the last 24 hours.
The slices of each pie represent the percent of the four types of outages: nodes, interfaces, services, and management.

The next section provides outage details in two tables: the first shows the current outages by nodes and the second shows the resolved outages by node.

The third section is a stacked bar chart that shows the current outages stacked on the resolved outages over the last 24 hours.

> The fourth section provides an overview of notifications over the past 24 hours.
> It includes a pie chart and table showing the number of notifications by the media type.

Reviewer: The first sentence is sufficient for this doc.
```suggestion
The fourth section provides an overview of notifications over the past 24 hours.
```

> // QUESTION: I still don't know what the slices are, there was nothing in this section of the sample report.  I forgot to ask Mark to send me a copy of this report using real data.
> 
> // QUESTION: I am not sure what "media type"

Reviewer: Media type is the method of how the notice was sent - email, chat, etc
```suggestion
```

This report provides a global overview of outages, notifications, and events in the last 24 hours.

> In the Outage Overview section, there are two pie charts, one that shows the current outages and one that shows the outages resolved in the last 24 hours. The slices of each pie represent the percent of the four types of outages: nodes, interfaces, services, and management.

Reviewer 2: ```suggestion
In the Outage Overview section, there are two pie charts, one that shows the current outages and one that shows the outages resolved in the last 24 hours.
The slices of each pie represent the percent of the four types of outages: nodes, interfaces, services, and management.
```

> The next section provides outage details in two tables, the first shows the current outages by nodes and the second table shows the resolved outages by node.

Reviewer 2: ```suggestion
The next section provides outage details in two tables: the first shows the current outages by nodes and the second shows the resolved outages by node.
```
Comma is incorrect, so colon or second sentence.

The third section is a stacked bar chart that shows the current outages stacked on the resolved outages over the last 24 hours.

The fourth section provides an overview of notifications over the past 24 hours.
It includes a pie chart and table showing the number of notifications by the media type.

// QUESTION: I still don't know what the slices are, there was nothing in this section of the sample report.

// QUESTION: I am not sure what "media type"

The fifth section provides and overview of events over the last 24 hours.

> The first stacked bar chart shows, for each of the last 24 hours, the number of each of the 7 types of events: indeterminate, cleared, normal, warning, minor, major, and critical.

Reviewer 2: ```suggestion
The first stacked bar chart shows, for each of the last 24 hours, the number of each of the seven types of event: indeterminate, cleared, normal, warning, minor, major, and critical.
```

> The second stacked bar chart shoes the top 20 nodes with each type of event per node.

Reviewer 2: ```suggestion
The second stacked bar chart shows the top 20 nodes with each type of event per node.
```
